pkg/autodiscover/autodiscover_podset.go

```diff
@@ -86,12 +97,12 @@ func findStatefulSetByLabel(
 			options := metav1.ListOptions{}
 			options.LabelSelector = label
 			statefulSetClient := appClient.StatefulSets(ns)
-			dps, err := statefulSetClient.List(context.TODO(), options)
+			ss, err := statefulSetClient.List(context.TODO(), options)
```

Comment:
```suggestion
			ss, err := statefulSetClient.List(context.TODO(), metav1.GetOptions{})
```

Reply:
option is modified with label selector, so need the assignment there